src/styles/ClientInfoName.scss

```diff
@@ -0,0 +1,8 @@
+.ClientInfoName {
```

Comment:
Fix the name of the file from `ClientInfoName.scss` to `ClientInfo.scss`

Reply:
Yes I forgot I'm working on it. That was when I tried to use className to use a media query inside, it did not worked.

Reply:
Yeah, I think that if you're not doing anything inside that scss file you can remove it.